Themes/default/css/index.css

```diff
@@ -597,9 +597,9 @@ dl.settings {
 	padding: 5px;
 }
 dl.settings dt {
-	width: 56%;
+	width: 45%;
```

Comment:
From what I can tell, it is currently broken everywhere else, too. But I can change the percentages back if you prefer. Those were mostly just cosmetic.